engine/src/main/java/org/entando/entando/aps/system/services/page/PageService.java

```diff
@@ -278,6 +287,29 @@ public PageDto updatePage(String pageCode, PageRequest pageRequest) {
         }
     }
 
+    @Override
+    public PageDto updatePage(String pageCode, JsonNode jsonPatch) {
+        IPage oldPage = this.getPageManager().getDraftPage(pageCode);
+        if (null == oldPage) {
+            throw new RestRourceNotFoundException(null, "page", pageCode);
+        }
+
+
+
+        try {
+            PageDto pageDto = this.getDtoBuilder().convert(oldPage);
+            PageDto updatedPageDto = this.getJsonPatchService().applyPatch(jsonPatch, pageDto);
+            PageRequest updateRequest = new PageDtoToPageRequestConverter().convert(updatedPageDto);
+            return this.updatePage(pageCode, updateRequest);
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Use a logger here

Reply:
Should I throw the error anyway right?

Reply:
Maybe you can catch a more specific exception (like the `PatchException` that can be thrown by the `applyPatch()` method) and rethrow a `RestServerError`, like the other methods are doing.